kandinsky/fonts/code_points.h

```diff
@@ -150,209 +150,9 @@ uint32_t SimpleCodePoints[] = {
   0x2264,  // ≤ // LESS-THAN OR EQUAL TO
   0x2265,  // ≥ // GREATER-THAN OR EQUAL TO
   0xFFFD,  // � // REPLACEMENT CHARACTER
-  0x1d422, // 𝐢 // MATHEMATICAL BOLD SMALL I"
+  0x1d422, // 𝐢 // MATHEMATICAL BOLD SMALL I" (Problems with this character: the symbol is linked to the "character not found" symbol
```

Comment:
I think https://github.com/UpsilonNumworks/Upsilon/commit/24205a7e03528d413fe201b58096c245bd42091b fix your bug (in `kandinsky/include/kandinsky/font.h`)